reference-architecture/gce-cli/ansible/playbooks/roles/openshift-ansible-installer/tasks/main.yaml

```diff
@@ -0,0 +1,6 @@
+---
+- name: download openshift ansible installer from github
+  git:
+    repo: https://github.com/openshift/openshift-ansible.git
+    dest: '{{ playbook_dir }}/../../../../../openshift-ansible'
```

Comment:
I'm ok with this if we add support for it in the future. It does currently stink that you have to use a license for OCP on your deployer box.